### DISCUSSION TITLE
Hello World! 👋

### DISCUSSION THREAD

**Discussion by Lamparter** [RiversideValley/Emerald#27] [Team Posts]
Welcome to Emerald Developer Chat.

**Comment by developerx-official** [comment]
I didn't know it had chat haha
That's neat :>

**Comment by Lamparter** [comment]
> I didn't know it had chat haha
> That's neat :>

I integrated teams feature into the Depth organisation. It allows me to better control perms of people, but it also gives me this nice chat environment. So yeah, haha it is nice

**Comment by developerx-official** [comment]
That is really nice :3
